content/docs/gke/troubleshooting-gke.md

```diff
@@ -417,3 +417,53 @@ kubectl -n kubeflow create secret generic kubeflow-oauth \
        --from-literal=client_id=${CLIENT_ID} \
        --from-literal=client_secret=${CLIENT_SECRET}
 ```
+
+## Managed certificates can fail if service management API not enabled
+
+This section describes how to enable service management API to avoid managed certificates failure.
+
+To check your certificate: 
+1. Run the following command:
+
+```
```

Comment:
To make the list formatting work from here downwards, I think you'll need to indent the code block by 2 spaces.

Reply:
LGTM.
I added the spaces.